### ISSUE TITLE
Space between boxplots not overlapping 

### ISSUE THREAD

**Issue by bcuer** [njudd/ggrain#10]
Hello,

Thnak you for your remarkable package very useful 👍. 

I would like to reproduce a graph from the package's vignette using the following code: 

```
ggplot(iris, aes(1, Sepal.Width, fill = Species, color = Species)) +
    geom_rain(alpha = .5, rain.side = 'l',
              boxplot.args = list(color = "black", outlier.shape = NA),
              boxplot.args.pos = list(
                  position = ggpp::position_dodgenudge(x = .1), width = 0.1
              )) +
    theme_classic() +
    scale_fill_brewer(palette = 'Dark2') +
    scale_color_brewer(palette = 'Dark2') +
    guides(fill = 'none', color = 'none')
```
It produces the following plot: 

 ![Rplot](https://github.com/njudd/ggrain/assets/42000413/8e091291-48a4-4d06-b70a-b56219e7a581)

Unfortunately, the result differs significantly from what is shown in the vignette. Specifically, the boxplots appear to be widely spaced, unlike the compact arrangement depicted in the vignette. I am unsure of the parameter adjustments needed to address this discrepancy.

For your information, I used to work with R version 4.1.2, and the generated graphics were accurate. 
However, after recently updating to the latest R version (4.3.2), I've observed that certain types of graphs are no longer reproducible.

Thank you for any suggestions or improvements.
Benjamin

**Comment by lusemp** [comment]
Hi 

I experience the same issue while using the f2x2 notation. 

geom_rain(alpha = .5, rain.side = 'f2x2', id.long.var = "ID",
            violin.args = list(color = NA, alpha = .5), show.legend = FALSE)

Thanks for any ideas how to solve it. cheers, 
Lukas

**Comment by njudd** [comment]
Are you guys using Windows? I can't seem to replicate the error with R 4.3.2 on mac
@jorvlan can you see if you can replicate this issue (even just on mac)?

**Comment by lusemp** [comment]
I use mac and R 4.3.2 and could replicate the issue. It only started occurring after I updated multiple packages. 

<img width="1440" alt="Screenshot 2024-01-19 at 07 21 08" src="https://github.com/njudd/ggrain/assets/157060578/ac68030a-8c07-473f-b42d-462a16165992">

**Comment by njudd** [comment]
Thanks for isolating it, just re-opening the issue until I fix it! 
Will figure out what is going on with the new version of gppp :)

**Comment by njudd** [comment]
The latest version of ggrain 0.0.4 _should_ solve this issue. I will keep it open until I update it on CRAN.

To get the latest version:
```
if (!require(remotes)) {
    install.packages("remotes")
}
remotes::install_github('njudd/ggrain')
library(ggrain)
```

The solution is quite simple, width needs to be given twice now with ggpp 0.5.6. 

_previously:_
boxplot.args.pos = list(position = ggpp::position_dodgenudge(x = .1), width = 0.1)
_ggpp 0.5.6:_
boxplot.args.pos = list(position = ggpp::position_dodgenudge(x = .1**, width = 0.1**), width = 0.1)

Thanks a million for identifying that it was ggpp, saved a bunch of time!

**Comment by njudd** [comment]
Fixed in version 0.0.4; when installing ggrain choose 1 'update all packages'.
